examples/controller-example/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-example
+  namespace: agones-system
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: controller-example
+  template:
+    metadata:
+      labels:
+        app: controller-example
+    spec:
+      serviceAccountName: default
```

Comment:
Might need some RBAC permissions to apply labels? 🤔 but you aren't getting permission errors, so maybe its fine?

Reply:
I haven't seen any RBAC permissions issue. Will check it out.